fftw-src/.gitattributes

```diff
@@ -0,0 +1 @@
+fftw-*/**/* linguist-generated=true
```

Comment:
From the documentation you linked above (emphasis mine)
> 1. Unless the .gitattributes file already exists, create a .gitattributes file **in the root of the repository**.
That would suggest this file needs to be one hierarchy level up. Are you sure it will work like this?

Reply:
AFAIK, .gitignore and .gitattribute work wherever it is placed. It seems to works on GitHub
![image](https://user-images.githubusercontent.com/1238153/85396745-39070900-b58d-11ea-8047-a2328a01a5a2.png)
https://github.com/rust-math/fftw/pull/88/commits/1e7366efe2f439ce1cfe2e5211969aaaa25724ff

But it would be better to put it on the top of repository. I've moved it.